thornodes/alerting.md

```diff
@@ -4,6 +4,16 @@ description: Telegram Notification Service
 
 # Alerting
 
+## Community alert channels
+
+To listen for update announcements, join the following channels: 
+* Telegram **THORNode Announcements** [https://t.me/thornode_ann](https://t.me/thornode_ann)
+* Discord **THORChain Community Devs** [https://discord.gg/kvZhpEtHAw](https://discord.gg/kvZhpEtHAw)
```

Comment:
[nitpick] For Discord: especially `#thornode-mccn` /` #thornode-mctn` channels

Reply:
Updated